actions/image-build/build.sh

```diff
@@ -54,12 +54,15 @@ else
 fi
 
 #Step 2: Configure
-
+export INSTALL_PATH=install
```

Comment:
Is this consistent with `CMAKE_PREFIX_PATH` and is there any reason not to put this path in the toolchain file?

Reply:
> Is this consistent with `CMAKE_PREFIX_PATH` and is there any reason not to put this path in the toolchain file?

Thanks for pointing out this! I did not realize that the installation directory has been set in the toolchain file generated by build.sh. I can use `CMAKE_PREFIX_PATH` instead of `INSTALL_PATH` here.

Reply:
> Is this consistent with `CMAKE_PREFIX_PATH` and is there any reason not to put this path in the toolchain file?

After some reconsideration, I think I want to continue to use INSTALL_PATH instead of CMAKE_PREFIX_PATH in the toolchain file. I have two reasons: 
1. CMAKE_PREFIX_PATH may contain other paths (the line `echo "set(CMAKE_PREFIX_PATH $(pwd)/install)"` will definitely be modified in the future), e. g., the path of the installation directory of libint2;
2. Not sure if I put the definition of INSTALL_PATH in the toolchain file, the command line `${cmake_command} -GNinja -H. -Bbuild -DCMAKE_TOOLCHAIN_FILE="${toolchain_file}" -DCMAKE_INSTALL_PREFIX=${INSTALL_PATH}` would recognize `${INSTALL_PATH}` or not.